### DISCUSSION TITLE
Lab Event Mapping from Local Itemid to LOINC

### DISCUSSION THREAD

**Discussion by tianranzhang** [MIT-LCP/mimic-code#1138] [MIMIC-IV]
1. I am currently writing a manuscript that involved MIMIC IV (version 0.4) and discovered that in the newly released lab event mapping file d_labitems.csv from version 1.0 several local itemid has changed, while other information in the same row remained unchanged. For example, for [bilirubin, urine, chemistry], its itemid was 51961 in MIMIC IV version 0.4 versus 51966 in version 1.0. How should we understand this change? Was version 0.4 itemid inaccurate, or was it due to an update in the internal lab item coding system between version 0.4 and version 1.0?

2. It seems that only 267/1625 labs from MIMIC IV's  (ver. 0.4) d_labitems table have LOINC codes available - while the documentation said 'most concepts in this table have been mapped to LOINC codes. In MIMIC-III, 586 out of 753 lab items were mapped to LOINC codes. Is the itemid from MIMIC-iii transferrable to MIMIC-IV data? How come there are more labs available in MIMIC-iv but fewer LOINC mappings available than MIMIC-iii?

Thanks.

**Comment by SichengH** [comment]
1, please using MIMIC-IV v1.0. v0.4 was a testing version. 

2, itemid in MIMIC-III start with 2xx are the same as in MIMIC-IV. And MIMIC-IV contains more itemids. The LOINC information in MIMIC-IV needs to be updated. We are aware of this issue. You can do a table join between MIMIC-III and IV to fill those LOINC codes, but always good to have some sanity check after.

**Reply by tianranzhang** [reply]
Hello Sicheng,
Thanks for the useful info. I have also noticed that MIMIC-IV has a microbiologyevents table and no itemid to LOINC mapping was provided in the lab item mapping file. Is there a way to obtain that mapping somewhere? I noticed the MIMIC-OMOP effort [https://github.com/MIT-LCP/mimic-omop] that standardized concepts in MIMIC-III, I wonder if a itemid to LOINC is possible through that. Any suggestions will be appreciated! Thanks!
